### ISSUE TITLE
How do I run this?

### ISSUE THREAD

**Issue by FRYVCA** [jaketae/storyteller#10]
I cloned as instructed.
Installed python as instructed.
Did update as instructed, but some files were not updated
"ERROR: Could not find a version that satisfies the requirement tts (from storyteller[dev]) (from versions: none)
ERROR: No matching distribution found for tts"

After that I am completely lost as to how I run it? 
Tried $storyteller, but error
Tried $ python pyproject.toml
Tried all other combinations, but because lack of expertise, unable to run it. Can you please give instructions for someone not sound with programming, a Layman explanation.

**Comment by jaketae** [comment]
Hello, thanks for opening this issue. I've just released the package on PyPI, so hopefully this makes it easier to run Story Teller. 

Could you do the following?

1. Install PyTorch based on instructions [here](https://pytorch.org/get-started/locally/). Make sure to choose the right OS, and check whether you have a CUDA-enabled GPU. (If this doesn't make sense to you, you can skip.)
2. In the terminal, run `pip install storyteller-core`. This should start installing the necessary packages to run Story Teller.
3. Once installation is done, simply type `storyteller` from the terminal. 

If you get an error, can you copy all the error messages and post them here? This will help with debugging. Thanks!

**Comment by chrisesmann** [comment]
Hi @jaketae I'm facing an error when running `pip install storyteller-core`
Getting the following error related to TTS.

      check.warn(importable)
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]
  
      note: This error originates from a subprocess, and is likely not a problem with pip.
      ERROR: Failed building wheel for tts
      Failed to build tts
      ERROR: Could not build wheels for tts, which is required to install pyproject.toml-based projects

**Comment by gvargas9** [comment]
Pip install tts

**Comment by jaketae** [comment]
Hello, thanks for reporting these issues.

[`tts`](https://pypi.org/project/TTS/) requires Python `>=3.7, <3.11`. So if you're on Python 3.11 or above, `pip` will fail to find a version of `tts` that matches your environment. I would recommend that you create a virtual environment with one of the allowed Python versions if this is the case.

I'm not a huge fan of their Python version requirement, but that's what it looks like at the moment. As @gvargas9 suggests, I would also consider running `pip install TTS` first, then installing `storyteller-core` to see if the issue goes away.

Let me know if this helps!

**Comment by cvDreamer** [comment]
I followed the instruction to run "pip install storyteller-core" and found the following error:

error: subprocess-exited-with-error . python setup.py egg_info did not run successfully. exit code: 1,

when i try to intall myself by running 'pip install .', i got the following error: 

'ModuleNotFoundError: No module named 'poetry'

I tried 'pip install poetry' but did not help.

My python version is 3.10.9 which should be satisfying the requirements.
